.github/actions/linux-build/action.yml

```diff
@@ -73,8 +77,8 @@ runs:
             export CC=/usr/bin/gcc-9
             export CXX=/usr/bin/g++-9
           else        
-            export CC=/usr/bin/gcc-10
-            export CXX=/usr/bin/g++-10
+            export CC=/usr/bin/gcc-11
+            export CXX=/usr/bin/g++-11
```

Comment:
Should we use variables for the version numbers now, or do we have a follow up issue for that?

Reply:
I think @MaxBetzDLR already did that. Didn't you show me that?

Reply:
I see neither variables nor an issue related to this, what are you referring to?

Reply:
Talk to him please.

Reply:
Git actions do not support a good way for variables. The possible implementations do not seem sensible to me.